Tests/SourceKitLSPTests/DefinitionTests.swift

```diff
@@ -543,62 +543,27 @@ class DefinitionTests: XCTestCase {
     )
   }
 
-  func testJumpToDefinitionOnProtocolImplementationJumpsToRequirement() async throws {
+  func testJumpToSatisfiedProtocolRequirementInExtension() async throws {
     let project = try await IndexedSingleSwiftFileTestProject(
       """
       protocol TestProtocol {
         func 1️⃣doThing()
       }
 
-      struct TestImpl: TestProtocol {
-        func 2️⃣do3️⃣Thing() { }
-      }
-      """
-    )
-
-    let definitionFromBaseName = try await project.testClient.send(
-      DefinitionRequest(textDocument: TextDocumentIdentifier(project.fileURI), position: project.positions["2️⃣"])
-    )
-    XCTAssertEqual(
-      definitionFromBaseName,
-      .locations([Location(uri: project.fileURI, range: Range(project.positions["1️⃣"]))])
-    )
-
-    let definitionFromInsideBaseName = try await project.testClient.send(
-      DefinitionRequest(textDocument: TextDocumentIdentifier(project.fileURI), position: project.positions["3️⃣"])
-    )
-    XCTAssertEqual(
-      definitionFromInsideBaseName,
-      .locations([Location(uri: project.fileURI, range: Range(project.positions["1️⃣"]))])
-    )
-  }
-
-  func testJumpToDefinitionOnProtocolImplementationShowsAllFulfilledRequirements() async throws {
-    let project = try await IndexedSingleSwiftFileTestProject(
-      """
-      protocol TestProtocol {
-        func 1️⃣doThing()
-      }
-
-      protocol OtherProtocol {
-        func 2️⃣doThing()
-      }
```

Comment:
Is it not worth keeping these tests?

Reply:
Thank you for catching. I think I accidentally lost them during a rebase.